internal/service/cloud/logic.go

```diff
@@ -849,3 +858,51 @@ func GetAllExportFiles(ctx context.Context, cID string, eID string, d TiDBCloudC
 	}
 	return items, nil
 }
+
+func RetrieveAuthorizedNetworks(ctx context.Context, clusterID string, d TiDBCloudClient) ([]cluster.EndpointsPublicAuthorizedNetwork, error) {
```

Comment:
need this? it's just get cluster with basic view and get fields

Reply:
a lot of places use the same logic, I think create a function is better